templates/mass_eigenstates.cpp.in

```diff
@@ -462,18 +462,36 @@ double CLASSNAME::G0(double p, double m1, double m2) const noexcept
    ).real();
 }
 
-/// derivative of B0(p^2, ma^2, mb^2) w.r.t. p^2
-double CLASSNAME::DB0(double p2, double m2a, double m2b) const noexcept
+double CLASSNAME::DH0(double p, double m1, double m2) const noexcept
 {
-   return softsusy::db0(p2, m2a, m2b);
+   const double scl2 = Sqr(get_scale());
+   return 4.0*Loop_library::get().DB00(p, m1, m2, scl2).real() + CLASSNAME::DG0(p, m1, m2);
+}
+
+double CLASSNAME::DF0(double p, double m1, double m2) const noexcept
```

Comment:
Same here, please add a brief comment.

Reply:
Done in 46963d81122636e42cb7c82d7f26193087090e7d.